static/styles/zulip.css

```diff
@@ -2623,6 +2623,17 @@ div.floating_recipient {
     margin-right: 10px;
 }
 
+.stream_sub_unsub_button,
+.empty_feed_join {
```

Comment:
In place of `.empty_feed_join`, I have introduced a button `.stream_sub_unsub_button` with same css.